app/javascript/src/components/Dashboard/Notes/Pane/Form/index.jsx

```diff
@@ -1,26 +1,29 @@
 import React from "react";
 
 import { Formik, Form as FormikForm } from "formik";
-import { Button, Pane } from "neetoui";
+import moment from "moment";
+import { Button, Pane, Toastr } from "neetoui";
 import { Input, Textarea, Select } from "neetoui/formik";
 
 import { TAGS } from "./constants";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../../constants";
 
 const Form = ({ onClose, note, isEdit, setShowPane, notes, setNotes }) => {
-  const handleSubmit = ({ title, description, assignedContact, tag }) => {
+  const handleSubmit = values => {
     const updatedNotes = [
       ...notes,
       {
-        title,
-        description,
-        assignedContactId: assignedContact.value,
-        tag: tag.value,
+        title: values.title,
+        description: values.description,
+        assignedContact: { ...values.assignedContact },
+        tag: { ...values.tag },
```

Comment:
Can you explain to me why we are spreading and creating a new object here?

Reply:
This was originally

`{
        title,
        description,
        assignedContact,
        tag,
        createdAt: dayjs(),
      }`
      
 For the mentioned error in the below comment, I was changing a few values and adding a few to see where exactly is the error. I kept it like it until adding the delete feature.
 
 I changed it in the commit related to delete feature.